adapter/redisquota/driver.go

```diff
@@ -37,7 +39,9 @@ type response struct {
 
 // Get is to get a connection from the connection pool.
 func (cp *connPool) get() (*connection, error) {
+	cp.Lock()
```

Comment:
@ZackButcher made some changes here in the last couple of days, so there might be something wrong with the config or the aspect manager. This needs debugging...